leaderboards/grade_leaderboard.py

```diff
@@ -27,7 +27,8 @@ def normalize_id(key):
 @XBlock.needs("i18n")
 class GradeLeaderboardXBlock(LeaderboardXBlock):
     STUDENT_VIEW_TEMPLATE = "grade_leaderboard.html"
-    GRADE_SOURCES = (EdxLmsGradeSource, MockGradeSource)  # Ordered list of class types that know how to get student grades
+    # Ordered list of class types that know how to get student grades
+    GRADE_SOURCES = (EdxLmsGradeSource, MockGradeSource)
```

Comment:
@bradenmacdonald do I understand right that in case `MockGradeSource` is used only for tests? If that's so, it might make sense to not use it as a fallback for `EdxLmsGradeSource`, but inject into `GradeLeaderboardXBlock` on-demand, i.e. in tests. Something like:

    @property
    def grade_source(self):
        if self._grade_source is None:
            self._grade_source = EdxLmsGradeSource()
        return self._grade_source

    @grade_source.setter
    def grade_source(self, value):
        # optional isinstance or hasattr checks to ensure correct interface
        self._grade_source = value

Essentially it's a property dependency injection with a default.